components/CtaDownload/src/stories/bem.stories.mdx

```diff
@@ -153,6 +156,7 @@ import readme from "../../README.md";
       href="https://source.unsplash.com/800x800/?file"
       download="pretty-file-name-without-spaces"
       type="application/pdf"
+      aria-label="Download: Verslag dienstreis naar Delhi (PDF, 126,5 kB)"
```

Comment:
Ik snap dat het maar een voorbeeld is, maar het lijkt me handiger om geen decimalen te hebben voor download size. Minder risico op verwarring voor screen readers.

Reply:
just following design ;-)